git/git_test.go

```diff
@@ -29,7 +29,10 @@ import (
 const CookedCommitID = "4e991b55e3d58b9c3137a791a9986ed9c5069697"
 
 func TestGit(t *testing.T) {
-	git, err := git.NewWrapper(test.Username, test.Email)
+	git, err := git.WithConfig(git.Config{
```

Comment:
I tested creating the cnofig with no username/email and it also works, do we really need them ?

Reply:
we need them for any write operation in the repo's index objects (git add, git commit, etc).

Reply:
Ah OK, but on this test none of that happens and the test works without it, so maybe here we could remove this ? :thinking:

Reply:
I think so.